utils/license.go

```diff
@@ -119,7 +123,13 @@ func GetAndValidateLicenseFileFromDisk(location string) (*model.License, []byte)
 		mlog.Error("Found license key at %v but it appears to be invalid.", mlog.String("filename", fileName))
 		return nil, nil
 	}
-	return model.LicenseFromJson(strings.NewReader(licenseStr)), licenseBytes
+
+	var license model.License
+	if jsonErr := json.Unmarshal([]byte(licenseStr), &license); jsonErr != nil {
```

Comment:
Shouldn't we do an early return here?

Reply:
Sure, I was keeping the semantic unchanged but it makes sense to throw an error.